index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="nl">
+<head>
+    <meta charset="UTF-8">
+    <title>COVID-19 Notificatie App Status</title>
+    <link href="css/main.css" rel="stylesheet">
+    <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon"/>
+</head>
+<body>
+
+<img src="img/logo.svg" alt="Logo Rijksoverheid" class="logo" />
+
+<h1>COVID-19 notificatie app status</h1>
+
+<article>
+    <p>Hier geven we graag uitleg over de vragen en status van de app</p>
+
+    <details class="checked">
+
+        <summary>
+            <h2>De overheid kan nooit bij mijn contacten</h2>
+            <span>De gegevens worden alleen op je eigen telefoon opgeslagen.</span>
+        </summary>
+
+        <p>
+            (hier tekst met link naar explainer video hoe het decentrale protocol werkt)
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/D__UaR5MQao" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </p>
+    </details>
+
+    <details class="in-progress">
+        <summary>
+            <h2>Iedereen kan de werking van de app controleren</h2>
+            <span>De broncode wordt binnenkort openbaar</span>
+        </summary>
+
+    </details>
+
+    <details class="in-progress">
+        <summary>
+            <h2>De app is gebruiksvriendelijk voor iedereen</h2>
+        </summary>
+
+        <div>
+            <ul>
+                <li>De app wordt getest met mensen met een laag taalniveau</li>
+                <li>De app wordt getest met mensen met een visuele etc handicap</li>
+            </ul>
+        </div>
+    </details>
+
+
+    <details class="in-progress">
+        <summary>
+            <h2>Experts geven advies over de werking van de app</h2>
+            <span>Er is een adviescommissie</span>
+        </summary>
+    </details>
+
+
+    <details class="in-progress">
+        <summary>
+            <h2>De app is veilig</h2>
+            <span>Er is een adviescommissie</span>
+        </summary>
+
+        <div>
+            <ul>
+                <li>controle door NCSC aan de hand van een set publieke criteria waarover publiek gerapoprteerd zal worden</li>
+                <li>Open source / volledig inzicht in de applicatie en de server code; in principe kan een ieder zelf zijn of haar eigen versie van de app bouwen en (binnen de door de telefoon leveranciers gestelde grenzen) verifiëren dat deze app identiek is aan de gedistribueerde app en deze testen. Inclusief een process om gevonden problemen te rapporteren.</li>
+                <li>Standaard zaken als pentest, security audit (met publieke rapporten).</li>
+                <li>Een "Responsible Disclosure" policy met voldoende capaciteit, budget en kunde om gevonden en geporteerde problemen na eerste release aan te pakken</li>
```

Comment:
```suggestion
                <li>Een "Responsible Disclosure" policy met voldoende capaciteit, budget en kunde om gevonden en gerapporteerde problemen na eerste release aan te pakken</li>
```

? Of daadwerkelijk ‘geporteerd’?